src/hooks/useLoadProposalFromQuery.ts

```diff
@@ -5,17 +5,20 @@
 export const queryKeys = {
   targets: "targets",
   calldatas: "calldatas",
-  values: "values"
-};
+  values: "values",
+  nonce: "nonce",
+} as const;
 
 export const useLoadProposalFromQuery = () => {
   const [proposal, setProposal] = useState<undefined | Proposal>();
```

Comment:
were going to have a state sync issue here - proposal is now saved here, and in another place where the context is.